src/OrchardCore/OrchardCore.Abstractions/Localization/DataLocalizedString.cs

```diff
@@ -0,0 +1,17 @@
+namespace OrchardCore.Localization
+{
+    public class DataLocalizedString
+    {
+        public DataLocalizedString(string context, string name)
+        {
+            Context = context;
+            Name = name;
+        }
+
+        public string Context { get; }
```

Comment:
This should be a `LocalizedString` so that the UI can be displayed in multiple languages